docs/tutorials/data-loading/remote-files.md

```diff
@@ -15,82 +13,190 @@ You need the following software installed:
 * Git
 * Docker
 * docker-compose
-* Node.js
+* Node.js (optional)
+* [corectl](https://github.com/qlik-oss/corectl#download) (optional)
 
 !!! Note
     Shell commands should be run in a Bash shell.
     If you are using Windows, we recommend using Git Bash.
 
-## The File Connectivity Service
-
-The File Connectivity Service is an example of a service that you can use to load data from remote files
-in a Qlik Core stack. You can configure this service to provide built-in data connectivity to connect to
-[OAuth 2.0](https://oauth.net/2/)-protected data sources like Dropbox, OneDrive, and Google Drive.
+## The gRPC S3 File Connector
 
-The data connection service in this tutorial works by defining a unique HTTP endpoint for each registered
-connection provider. The Qlik Associative Engine can then access different data sources by making calls to the
-service-defined HTTP endpoints.
+The S3 File Connector example is a connector built to display how to be able to download and upload files
+to and from an S3 bucket in AWS.
+The connector is built in Node.js and uses the gRPC protocol to stream data to and from the Qlik Associative Engine.
 
 ## Setup
 
 To run the example code, clone the
-[qlik-oss/core-file-connectivity-service](https://github.com/qlik-oss/core-file-connectivity-service) Git repository.
+[qlik-oss/core-grpc-s3-file-connector](https://github.com/qlik-oss/core-grpc-s3-file-connector) Git repository.
 Before you continue, look at the documentation to get familiar with the content and structure of
 the repository.
 
 !!!Note
     In your shell, make sure current directory is at the repository root.
 
-First, install the dependencies:
+The connector example repo provides a `docker-compose.yml` file
```

Comment:
Link to the yaml file instead (I guess we can predict where it's gonna be)? The inline file below will get outdated real quick, better to link to source that we know should work for them.

Reply:
I agree that it will get outdated but it feels more readable when having the content inline instead of having to open it in another tab.

Reply:
* Added link to the yml file
* Removed version from the inline code

The only remaining stuff in the inline sample that might change going forward is the env variables, but then we need to update the tutorial anyway.